File: src/main/java/me/miki/shindo/utils/animation/normal/easing/EaseLiner.java
package me.miki.shindo.utils.animation.normal.easing;

import me.miki.shindo.utils.animation.normal.Animation;

public class EaseLiner extends Animation {

    public EaseLiner(int ms, double endPoint) {
        super(ms, endPoint);
        this.reset();
    }

    @Override
    protected double getEquation(double x) {
        return x / duration;
    }
}

File: src/main/kotlin/me/miki/shindo/utils/animation/normal/easing/EaseLiner.kt
package me.miki.shindo.utils.animation.normal.easing

import me.miki.shindo.utils.animation.normal.Animation

class EaseLiner(ms: Int, endPoint: Double) : Animation(ms, endPoint) {

    init {
        this.reset()
    }

    override fun getEquation(x: Double): Double {
        return x / duration
    }
}
